cli/internal/cmd/miniup.go

```diff
@@ -202,11 +202,12 @@ func (m *miniUpCmd) prepareConfig(cmd *cobra.Command, fileHandler file.Handler)
 	_, err = fileHandler.Stat(constants.ConfigFilename)
 	if err == nil {
 		// config already exists, prompt user to overwrite
-		cmd.PrintErrln("A config file already exists in the current workspace.")
+		cmd.PrintErrln("A config file already exists in the current workspace. If you know what you are doing, you can use the --config flag to use your own config.")
```

Comment:
```suggestion
		cmd.PrintErrln("A config file already exists in the current workspace. Delete the config or use --config to specify another config file.")
```

I am not set on my particular wording. But I would remove the "If you know what you are doing, " part. It seems a bit confrontative to me :D

Reply:
I just wanted to emphasize that this is and advanced feature that should be handled with care, but yes, maybe it could sound a bit _confrontative_ so I'll remove this :)